src/js/components/Pagination/PaginationSummary.js

```diff
@@ -0,0 +1,23 @@
+import React, { useContext } from 'react';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { MessageContext } from '../../contexts/MessageContext';
+
+export const PaginationSummary = ({ messages, numberItems, page, step }) => {
+  const { format: formatMessage } = useContext(MessageContext);
+  return (
+    <Box flex="grow">
+      <Text>
+        {formatMessage({
+          id: 'pagination.summary',
+          messages,
+          values: {
+            firstValue: `${(page - 1) * step + 1}`,
```

Comment:
@taysea opening to naming ideas here

Reply:
Cool, dropped a suggestion on default.json file